frappe/handler.py

```diff
@@ -43,15 +43,12 @@ def handle():
 
 def execute_cmd(cmd, from_async=False):
 	"""execute a request as python module"""
-	for hook in frappe.get_hooks("override_whitelisted_methods", {}).get(cmd, []):
-		# override using the first hook
-		cmd = hook
-		break
-
 	# via server script
 	if run_server_script_api(cmd):
 		return None
 
+	cmd = get_overridden_method(cmd)
+
```

Comment:
This changes the order of how whitelisted APIs are resolved. Update the docs for the same!

Reply:
This doesn't needs to be changed, so I'll revert this.